lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -74,71 +76,54 @@ class SubjectStub < Cop
         PATTERN
 
         def on_block(node)
+          # process only describe/context/... example groups
           return unless example_group?(node)
 
-          find_subject_stub(node) do |stub|
+          # skip already processed example group
+          # it's processed if is nested in one of the processed example groups
+          return unless (processed_example_groups & node.ancestors).empty?
+
+          # add example group to already processed
+          processed_example_groups << node
```

Comment:
Is there any evidence that `on_block` can be called more than once for the same example group?

Reply:
I believe no. The goal of tracking nodes is to prevent handling of blocks nested into already processed outer blocks.

Reply:
👍